core/src/main/java/com/google/instrumentation/trace/TraceExporter.java

```diff
@@ -172,6 +171,99 @@ static TraceExporter getNoopTraceExporter() {
      */
     public abstract void unregisterSpanNamesForCollection(Collection<String> spanNames);
 
+    /**
+     * The latency buckets boundaries. Samples based on latency for successful spans (the status of
+     * the span has a canonical code different than {@link CanonicalCode#OK}) are collected in one
+     * of these latency buckets.
+     */
+    public enum LatencyBucketsBoundaries {
```

Comment:
by sharing with UI, do you mean sharing by convention or literally writing this to json or such?